### PR TITLE
Fix the broken link of "Getting Started"

### LINKED ISSUE #1386: Broken link of "Getting Started"
### Describe the bug

When clicking the below "Getting Started" link,
getting-started

404 is displayed.
getting-started2

### Version

16.1.1

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by y-tabata**
Closes https://github.com/keycloak/keycloak-documentation/issues/1386

### CODE REVIEW

**Review by tnorimat — DISMISSED** [review]
LGTM

File: topics/templates/document-attributes-community.adoc [hunk 1]
```diff
@@ -68,8 +68,8 @@ endif::[]
 :developerguide_jsproviders_link: {developerguide_link}#_script_providers
 :gettingstarted_name: Getting Started Guide
 :gettingstarted_name_short: Getting Started
-:gettingstarted_link: https://www.keycloak.org/getting-started
-:gettingstarted_link_latest: https://www.keycloak.org/getting-started
+:gettingstarted_link: https://www.keycloak.org/guides
+:gettingstarted_link_latest: https://www.keycloak.org/guides
```
Comment (stianst) [inline topics/templates/document-attributes-community.adoc]:
```suggestion
:gettingstarted_link: https://www.keycloak.org/guides#getting-started
:gettingstarted_link_latest: https://www.keycloak.org/guides#getting-started
```

Comment (stianst) [reply topics/templates/document-attributes-community.adoc]: Adding the anchor for getting-started so it jumps to the correct section.

Comment (y-tabata) [reply topics/templates/document-attributes-community.adoc]: @stianst Thanks. I added the anchor.

**Review by stianst — APPROVED** [review]

**Pull request merged**